photonjs/README.md

```diff
@@ -162,6 +166,131 @@ type Profile = {
   - Graph selection: Travesal vs operate
   - ...
 
+## Constructor
+```ts
+
+export type LogLevel = 'INFO' | 'WARN' | 'QUERY' 
+
+export type LogOption = LogLevel | {
+  level: LogLevel
+  /**
+   * @default 'stdout'
+   */
+  emit?: 'event' | 'stdout'
+}
+
+export interface PhotonOptions {
+  datasources?: Datasources
+
+  /**
+   * @default false
+   */
+  log?: boolean | LogOption[]
+
+  /**
+   * You probably don't want to use this. \`__internal\` is used by internal tooling.
+   */
+  __internal?: {
+    debug?: boolean
+    hooks?: Hooks
+    engine?: {
+      cwd?: string
+      binaryPath?: string
+    }
+  }
+}
+```
+
+## Debugging & Logging
+
+### Debugging
```

Comment:
Please switch the order of debugging and logging in the spec. Logging is user facing and should come first, debugging is more low level and used less often.